tests/integration/test_integration.py

```diff
@@ -0,0 +1,96 @@
+import time
+import pytest
+import rethinkdb as r
+
+from bigchaindb import Bigchain
+
+
+@pytest.fixture
+def inputs(user_vk):
+    from bigchaindb.models import Transaction
+
+    b = Bigchain()
+
+    # create blocks with transactions for `USER` to spend
+    for block in range(4):
+        transactions = [
+            Transaction.create(
+                [b.me], [user_vk], metadata={'i': i}).sign([b.me_private])
+            for i in range(10)
+        ]
+        block = b.create_block(transactions)
+        b.write_block(block, durability='hard')
+
+
+@pytest.mark.usefixtures('processes')
+def test_fast_double_create(b, user_vk):
```

Comment:
I was wondering about a fast double transfer earlier today; something along the lines of:

```python
b.write_transaction(create_tx)

time.sleep(2)
b.write_transaction(transfer_tx_1)
b.write_transaction(transfer_tx_2)
```

where `transfer_tx_1` and `transfer_tx_2` may not be the same but need to be spending the `create_tx`. Not sure what we'd do here (on a single node, would they just keep getting re-added to the same block and fail?)...